src/versions/2.0/json.ts

```diff
@@ -145,22 +136,92 @@ export const METADATA_DEFINITIONS = {
     },
     required: ["license_number", "state"],
   },
+  affirmation: {
+    type: "object",
+    properties: {
+      affirmation: {
+        const:
+          "To the best of its knowledge and belief, the hospital has included all applicable standard charge information in accordance with the requirements of 45 CFR 180.50, and the information encoded is true, accurate, and complete as of the date indicated.",
+      },
+      confirm_affirmation: {
+        type: "boolean",
+      },
+    },
+    required: ["affirmation", "confirm_affirmation"],
+  },
+  modifier_information: {
+    type: "object",
+    properties: {
+      description: {
+        type: "string",
+      },
+      code: {
+        type: "string",
+      },
```

Comment:
Something I'm wondering about for these properties and other required string properties: do we need to add a `minLength: 1` keyword to each of them? Without that, an empty string will be considered valid.

Reply:
True. I think it is reasonable to add this requirement in.

Reply:
Added it in here: https://github.com/CMSgov/hpt-validator/commit/ec304e1ba8111812f24ff923bb132645f7e7340b